.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,33 +1,70 @@
-Make sure you have checked _all_ steps below.
 
-### Issue
+<!--
+Thanks for opening a pull request!
+If this is your first pull request you can find detailed information on how 
```

Comment:
If you're new to Parquet-Java, information on how to contribute can be found here: https://parquet.apache.org/docs/contribution-guidelines/contributing